react/game/src/redux/game/selectors.js

```diff
@@ -0,0 +1,24 @@
+import { concatHistory, calculateWinner } from '@utils/game';
```

Comment:
I'd rename this file `effects`. Selectors are something else that does enter in the scope of this card, it is other approach in an optional trello card